src/Oro/BugBundle/Entity/Issue.php

```diff
@@ -169,9 +169,20 @@ public function __construct()
      */
     public function __toString()
     {
-        return (string)$this->getCode();
+        return (string)$this->getCode().' '.(string)$this->getSummary();
```

Comment:
I think double typecast to string is excess here, concat will cast it anyway
In case your code and summary is tested you will get empty space, but it should be empty
Looks like this case is not covered in test